app/Models/Room.php

```diff
@@ -16,10 +16,6 @@ class Room extends Model
 
     protected $fillable = ['name', 'capacity'];
 
-    protected $with = ['users'];
-
-    protected $withCount = ['users'];
-
```

Comment:
I added these to optimize the queries... Why did you delete it?

Reply:
It doesn't work. If I type $room->users it returns the first user, not all of them. Just $room->users()->get() works for some reason